src/test/java/pl/ostek/internet_chat/service/MessageServiceTest.java

```diff
@@ -1,20 +1,35 @@
 package pl.ostek.internet_chat.service;
 
+import org.junit.jupiter.api.BeforeEach;
 import org.junit.jupiter.api.Test;
 import org.junit.jupiter.params.ParameterizedTest;
 import org.junit.jupiter.params.provider.*;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.mock.mockito.MockBean;
 import pl.ostek.internet_chat.exception.BlankMessageException;
 import pl.ostek.internet_chat.model.Message;
 import pl.ostek.internet_chat.repository.MessageRepository;
 
 import static org.assertj.core.api.Assertions.*;
+import static org.mockito.BDDMockito.given;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.verify;
 
+import java.util.Arrays;
 import java.util.List;
 import java.util.Map;
 
 class MessageServiceTest {
 
-    private MessageService messageService=new MessageService(new MessageRepository());
+    private MessageService messageService;
+
+    private MessageRepository messageRepository;
+
+    @BeforeEach
+    public void initAll(){
+        messageRepository=mock(MessageRepository.class);
```

Comment:
Czytelniej z mockito-extension i adnotacjami  np. https://solidsoft.wordpress.com/2018/03/27/convenient-mocking-in-mockito-with-junit-5-the-official-way/